src/sources/kubernetes_logs/mod.rs

```diff
@@ -123,50 +164,69 @@ pub struct Config {
 
     /// Ignore files with a data modification date older than the specified number of seconds.
     #[serde(default)]
+    #[configurable(metadata(docs::type_unit = "seconds"))]
+    #[configurable(metadata(docs::examples = 600))]
     ignore_older_secs: Option<u64>,
 
     /// Max amount of bytes to read from a single file before switching over
     /// to the next file.
+    ///
     /// This allows distributing the reads more or less evenly across
     /// the files.
+    #[configurable(metadata(docs::type_unit = "bytes"))]
     max_read_bytes: usize,
 
-    /// The maximum number of bytes a line can contain before being discarded. This protects
-    /// against malformed lines or tailing incorrect files.
+    /// The maximum number of bytes a line can contain before being discarded.
+    ///
+    /// This protects against malformed lines or tailing incorrect files.
+    #[configurable(metadata(docs::type_unit = "bytes"))]
     max_line_bytes: usize,
 
-    /// How many first lines in a file are used for fingerprinting.
+    /// The number of lines to read for generating the checksum.
+    ///
+    /// If your files share a common header that is not always a fixed size,
+    ///
+    /// If the file has less than this amount of lines, it won’t be read at all.
+    #[configurable(metadata(docs::type_unit = "lines"))]
     fingerprint_lines: usize,
 
     /// This value specifies not exactly the globbing, but interval
     /// between the polling the files to watch from the `paths_provider`.
```

Comment:
I think this sentence should be rewritten so that it makes sense.

Reply:
How's it now?

Addressed in 7b8d54c9b